### ISSUE TITLE
icon request: accessibility braille

### ISSUE THREAD

**Issue by MVprobr** [FortAwesome/Font-Awesome#8013]
![iconathon_braille_simple-black-gradient_512x512](https://cloud.githubusercontent.com/assets/3662026/11442689/ac0bfb7c-94fe-11e5-99ed-7c72b12ac1be.png)
![31435016-blindness-braille-writing-system-icon-set-stock-vector-braille](https://cloud.githubusercontent.com/assets/3662026/11442690/ac0c1d8c-94fe-11e5-953c-2a71e6f080de.jpg)
![12074316311517254337braille black svg med](https://cloud.githubusercontent.com/assets/3662026/11442691/ac11efb4-94fe-11e5-969e-f186d2aeccd0.png)

**Comment by caduvieira** [comment]
related #5753

**Comment by tagliala** [comment]
https://www.graphicartistsguild.org/tools_resources/downloadable-disability-access-symbols

**Comment by MVcodeartbr** [comment]
nice but this could be in inside fontawsome for us ux designer.

**Comment by codyn329** [comment]
+1
